contracts-periphery/Makefile

```diff
@@ -13,3 +13,4 @@ install :; $(INSTALL_CMD)
 test :; forge test
 test-gas :; forge test --match-path *.gas.t.sol
 snapshot-gas :; forge test --match-path *.gas.t.sol --gas-report > snapshot/.gas
+coverage :; forge coverage --report lcov && sed -i 's/SF:/SF:contracts-periphery\//gI' ./lcov.info
```

Comment:
If I run `yarn coverage` from the repo root, this part errors:

```
@umbra/contracts-periphery: Wrote LCOV report.
@umbra/contracts-periphery: sed: 1: "./lcov.info": invalid command code .
@umbra/contracts-periphery: make: *** [coverage] Error 1
@umbra/contracts-periphery: error Command failed with exit code 2.
@umbra/contracts-periphery: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run coverage exited 2 in '@umbra/contracts-periphery'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
error Command failed with exit code 2.
```

IIRC sed behaves differently on macOS (which I'm using) vs linux, wonder if that's the error? Seems like it thinks the `.` part of `./lcov.info` is a command?

Reply:
Ah that is probably the case, I tested on linux and I will resolve

Reply:
I think this should be resolved. I tested on mac and linux

Reply:
confirmed, thanks!